src/envoy/http/authn/http_filter_factory.cc

```diff
@@ -74,12 +80,48 @@ class AuthnFilterConfig : public NamedHttpFilterConfigFactory,
   std::string name() override { return kAuthnFactoryName; }
 
  private:
-  HttpFilterFactoryCb createFilter() {
+  // Convert istio-authn::jwt to jwt_auth::jwt in protobuf format.
+  void convertJwtAuthFormat(
+      istio::authentication::v1alpha1::Policy& policy,
```

Comment:
This function should simply take policy.Jwt instead of the whole policy. A policy can have more than one Jwt, in both peers and/or origins.

Reply:
Done.